src/dymaptic.GeoBlazor.Core/Scripts/definitions.d.ts

```diff
@@ -99,4 +99,22 @@ declare module "ArcGisDefinitions" {
         children: DotNetListItem[],
         actionSections: DotNetActionSection[][]
     }
+
```

Comment:
These are not used and should be deleted. Notice they are identical to `DotNetListItem`.

Reply:
removed and went with dotnetlistitem reference